.github/workflows/tests.yaml

```diff
@@ -3,13 +3,23 @@ on: [push, workflow_dispatch]
 jobs:
   tests:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./packages/ndk
     steps:
       - uses: actions/checkout@v1
-      - uses: subosito/flutter-action@v1
-      - run: flutter doctor
-      - run: flutter test --coverage
-      - run: flutter test ./example
+      - uses: dart-lang/setup-dart@v1
+      - run: dart pub get
+      - run: dart format --output=none .
+      - run: dart analyze --no-fatal-warnings
+      - run: dart test --coverage="coverage"
```

Comment:
can we use `melos run test:dart/flutter` here?
Right now there seems to be a dependency issue on rust_lib_ndk

Reply:
for now it is running tests only for ndk because of 
`working-directory: ./packages/ndk`
but to be decided how to run that on all packages once we HAVE tests for other packages besides ndk only

Reply:
I created a ticket